src/main/java/org/fcrepo/spec/testsuite/authz/WebACModes.java

```diff
@@ -132,23 +190,116 @@ public void writeAllowedDELETE(final String uri) {
+        final String aclUri = createAclForResource(resourceUri, "user-read-write.ttl", this.username);
+        //perform PATCH  to child resource as non-admin
+        doPatch(resourceUri, new Headers(new Header("Content-Type", APPLICATION_SPARQL_UPDATE)), body, false);
+
     }
 
     /**
-     * 5.0-M - Write access disallowed
+     * 5.0-M-1 - Write access disallowed on PUT
      *
      * @param uri of base container of Fedora server
      */
     @Test(groups = {"MUST"})
     @Parameters({"param1"})
-    public void writeDisallowed(final String uri) {
-        final TestInfo info = setupTest("5.0-M", "writeDisallowed",
-                "acl:Write gives access to a class of operations that can modify the resource. In a REST API " +
-                        "context, this would include PUT, POST, DELETE and PATCH. Its absence must prevent writes",
-                "https://fedora.info/2018/06/25/spec/#resource-authorization", ps);
+    public void writeDisallowedPut(final String uri) {
+        final TestInfo info = setupTest("5.0-M-1", "writeDisallowedPut",
+                                        "acl:Write gives access to PUT a resource. When not present, " +
+                                        "writes should be disallowed.",
+                                        "https://fedora.info/2018/06/25/spec/#resource-authorization", ps);
+
+        //create a resource
+        final String resourceUri = createResource(uri, info.getId());
+        final String aclUri = createAclForResource(resourceUri, "user-read-only.ttl", this.username);
+        //perform PUT  to child resource as non-admin with read only prives
+        final Response putResponse =
+            doPutUnverified(resourceUri + "/child1", new Headers(new Header("Content-Type", "text/plain")),
+                            "test", false);
+        //verify unauthorized
+        putResponse.then().statusCode(403);
+    }
+
+    /**
+     * 5.0-M - Write access disallowed POST
+     *
+     * @param uri of base container of Fedora server
+     */
+    @Test(groups = {"MUST"})
+    @Parameters({"param1"})
+    public void writeDisallowedPost(final String uri) {
+        final TestInfo info = setupTest("5.0-M-2", "writeDisallowedPost",
+                                        "acl:Write gives access to POST a resource. When not present, " +
+                                        "writes should be disallowed.",
+                                        "https://fedora.info/2018/06/25/spec/#resource-authorization", ps);
+        //create a resource
+        final String resourceUri = createResource(uri, info.getId());
+        final String aclUri = createAclForResource(resourceUri, "user-read-only.ttl", this.username);
+        //perform POST  to child resource as non-admin
+        final Response postResponse = doPostUnverified(resourceUri, new Headers(
+                                                           new Header("Slug", "child2")),
```

Comment:
As @bseeger [notes](https://github.com/fcrepo/Fedora-API-Test-Suite/pull/150#discussion_r212671999), we may want to begin getting away from the use of `slugs`s.